ports/servoshell/desktop/events_loop.rs

```diff
@@ -189,23 +189,3 @@ impl Drop for EventLoopGuard {
         CURRENT_EVENT_LOOP.with(|cell| cell.set(None));
     }
 }
-
-// Helper function to safely use the current event loop
```

Comment:
Can we also remove the CURRENT_EVENT_LOOP theead-local and EventLoopGuard type?

Reply:
Ah right. The guard type is obsolete after this change.